sklearn/inspection/_plot/tests/test_plot_partial_dependence.py

```diff
@@ -588,3 +588,44 @@ def test_partial_dependence_overwrite_labels(
             legend_text = ax.get_legend().get_texts()
             assert len(legend_text) == 1
             assert legend_text[0].get_text() == label
+
+
+@pytest.mark.filterwarnings("ignore:A Bunch will be returned")
+@pytest.mark.parametrize(
+    "line_kw, pd_line_kw, ice_lines_kw, expected_colors",
```

Comment:
We will also need a separate test to check that we raise the warning in the two expected case.
We usually used `pytest.warns` to catch those and check the recorder message.